ci/taos/plugins-good/pr-prebuild-clang.sh

```diff
@@ -56,10 +56,14 @@ function pr-prebuild-clang(){
     echo "[DEBUG] Files to be tested: $FILES_TO_BE_TESTED"
 
     # import clang configuration file
-    ln -sf ci/doc/.clang-format .clang-format
+    if [ ! -f ".clang-forma" ]; then
+	ln -sf ci/doc/.clang-format .clang-format
+    fi
 
     # run clang format checker
-    ${CLANG_COMMAND} -i $FILES_TO_BE_TESTED
+    for i in ${FILES_TO_BE_TESTED}; do
+	${CLANG_COMMAND} -i $i
```

Comment:
Yes. It seems that "-i" option can not handle multil files (e.g., a.cpp b.cpp c.cpp ....).
The variable $i must be a single file (e.g., test.cpp or *.cpp). This commit is reasonable to me. 

* Case study:  $ clang-format -i -style=WebKit *.cpp *.h